core/server/master/src/test/java/alluxio/master/journal/raft/RaftJournalSystemMetricsTest.java

```diff
@@ -59,66 +60,47 @@ public void metrics() throws Exception {
         .setFollowerInfo(RaftProtos.FollowerInfoProto.newBuilder()
             .setLeaderInfo(RaftProtos.ServerRpcProto.newBuilder()
                 .setId(RaftProtos.RaftPeerProto.newBuilder()
-                    .setId(ByteString.copyFromUtf8("localhost_19201")))))
+                    .setId(ByteString.copyFromUtf8("localhost_22201")))))
         .build();
 
-    Map<String, Long> sn1 = new HashMap<String, Long>() {
-      {
-        put("foo", 1L);
-      }
-    };
+    Map<String, Long> sn1 = ImmutableMap.of("DefaultBlockMaster", 1L, "DefaultMetaMaster", 2L);
     Mockito.doReturn(sn1).when(system).getCurrentSequenceNumbers();
-    system.startInternal();
+    system.start();
     Mockito.doReturn(null).when(system).getRaftRoleInfo();
     assertEquals(-1, getClusterLeaderIndex());
     assertEquals(-1, getMasterRoleId());
     assertEquals("WAITING_FOR_ELECTION", getClusterLeaderId());
     assertEquals(sn1, getMasterJournalSequenceNumbers(system));
 
-    Map<String, Long> sn2 = new HashMap<String, Long>() {
-      {
-        put("foo", 1L);
-        put("bar", 2L);
-      }
-    };
-    Mockito.doReturn(sn2).when(system).getCurrentSequenceNumbers();
     system.gainPrimacy();
     Mockito.doReturn(leaderInfo).when(system).getRaftRoleInfo();
     assertEquals(0, getClusterLeaderIndex());
     assertEquals(RaftProtos.RaftPeerRole.LEADER_VALUE, getMasterRoleId());
     assertEquals(system.getLocalPeerId().toString(), getClusterLeaderId());
-    assertEquals(sn2, getMasterJournalSequenceNumbers(system));
+    assertEquals(sn1, getMasterJournalSequenceNumbers(system));
```

Comment:
Could you add a bit more comments (in code) why the assertions will hold true? For example, "The journal sequence numbers should not change if the master transits from standby to primary"

Reply:
After a second thought, I think we shouldn't test `getCurrentSequenceNumbers()` change here.
The fact `losePrimacy()` will restart the `RaftJournalSystem` (and re-register metrics) is implementation detail.
And `RaftJournalSystem.mJournals` shouldn't be changed after `RaftJournalSystem.start()`.